src/sentry/static/sentry/app/views/organizationDiscover/sidebar/orderBy.jsx

```diff
@@ -0,0 +1,99 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import SelectControl from 'app/components/forms/selectControl';
+import {t} from 'app/locale';
+import styled from 'react-emotion';
+import {Flex} from 'grid-emotion';
+
+import {PlaceholderText, SidebarLabel} from '../styles';
+
+export default class OrderBy extends React.Component {
```

Comment:
Let's use `Orderby` instead of `OrderBy` here and elsewhere in this PR - for consistency across sentry we should treat it as a single word